src/Xz/LinuxXzBinaryLocator.php

```diff
@@ -4,7 +4,7 @@
 
 use rikudou\SkQrPayment\Exception\QrPaymentException;
 
-final class XzBinaryLocator implements XzBinaryLocatorInterface
+final class LinuxXzBinaryLocator implements XzBinaryLocatorInterface
```

Comment:
This is a backwards compatibility break, at least create the file ` src/Xz/XzBinaryLocator.php` and put something like this there:

```php
<?php

use rikudou\SkQrPayment\Xz\XzBinaryLocator;
use rikudou\SkQrPayment\Xz\LinuxXzBinaryLocator;

trigger_error(sprintf('The class "%s" is deprecated, please use "%s", XzBinaryLocator::class, LinuxXzBinaryLocator::class), E_USER_DEPRECATED);
class_alias(LinuxXzBinaryLocator::class, XzBinaryLocator::class);
```

There may be some mistakes in the code, I did it from memory without testing.